.env.example

```diff
@@ -15,23 +15,26 @@ REDIS_HOST=localhost
 REDIS_PORT=6379
 REDIS_PASSWORD=redis_password
 REDIS_SSL_ENABLED="false"
-AFM_URI="http://pagopa-gec-mock:3000/afm/node/calculator-service/v1/fees"
+AFM_URI="http://pagopa-gec-mock:8085/afm/calculator-service/v1/fees"
+AFM_URI_V2="http://pagopa-gec-mock:8085/afm/calculator-service/v2/fees"
 AFM_READ_TIMEOUT=10000
 AFM_CONNECTION_TIMEOUT=10000
 AFM_KEY=key
 SPRING_MAX_IN_MEM_SIZE=16777216
-NPG_URI="http://localhost/npg"
+NPG_URI="http://pagopa-npg-mock:8093/"
 NPG_READ_TIMEOUT=10000
 NPG_CONNECTION_TIMEOUT=10000
 NPG_API_KEY=key
 SESSION_URL_BASEPATH="https://localhost:1234"
 SESSION_URL_OUTCOME_SUFFIX="/esito"
 SESSION_URL_CANCEL_SUFFIX="/cancel"
+SESSION_URL_NOTIFICATION_URL="http://localhost:1234/{orderId}/{sessionToken}"
 NPG_SESSIONS_TTL=600
 OTEL_RESOURCE_ATTRIBUTES="service.name=pagopa-ecommerce-payment-methods-service,deployment.environment=dev"
 OTEL_EXPORTER_OTLP_ENDPOINT="http://otel-collector.elastic-system.svc:4317"
 OTEL_LOGS_EXPORTER="none"
 OTEL_TRACES_SAMPLER="always_on"
 OTEL_EXPORTER_OTLP_HEADERS="Authorization=Bearer aaa"
 NPG_NOTIFICATION_JWT_SECRET=ODMzNUZBNTZENDg3NTYyREUyNDhGNDdCRUZDNzI3NDMzMzQwNTFEREZGQ0MyQzA5Mjc1RjY2NTQ1NDk5MDMxNzU5NDc0NUVFMTdDMDhGNzk4Q0Q3RENFMEJBODE1NURDREExNEY2Mzk4QzFEMTU0NTExNjUyMEExMzMwMTdDMDk
-NPG_NOTIFICATION_JWT_VALIDITY_TIME=900
\ No newline at end of file
+NPG_NOTIFICATION_JWT_VALIDITY_TIME=900
+WARMUP_PAYMENT_METHOD_ID="e7058cac-5e1a-4002-8994-5bab31e9f385"
```

Comment:
please open a PR against ecommerce-local repo updating [payment methods service env variables](https://github.com/pagopa/pagopa-ecommerce-local/blob/main/pagopa-ecommerce-payment-methods-service/payment-methods-service.env)

Reply:
Also add the variable to `README.md` if you haven't already :)

Reply:
Thank you guys. Done.